src/sentry/models/email.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.db.models import Model, sane_repr
+from sentry.db.models import CIEmailField
+
+
+class Email(Model):
+    """
+    Email represents a unique email. Email settings (unsubscribe state) should be associated here.
+    UserEmail represents whether a given user account has access to that email.
+    """
+    __core__ = True
+
+    email = CIEmailField(_('email address'), unique=True)
+    date_added = models.DateTimeField(default=timezone.now)
```

Comment:
If we don't need an index on this now, we'll probably need to add one later if we try to do anything with this table regarding cleanup.

Reply:
I'm going to punt on this because I'm unsure when we'd need to do such a cleanup.